matrix/mat64/shadow.go

```diff
@@ -208,7 +208,7 @@ func rectanglesOverlap(off, aCols, bCols, stride int) bool {
 
 	// Flatten the shifted matrix column positions
 	// so a starts at 0, modulo the common stride.
-	const aFrom = 0
+	// const aFrom = 0
```

Comment:
The motivation to just comment it out was documentation. I expected that it would trigger some comments that would clarify what to do with this. I will remove it.